cccm-frontend/src/components/common/templateClientProfilePO.json

```diff
@@ -238,9 +238,44 @@
                                                         "input": false,
                                                         "tableView": false,
                                                         "components": [
+                                                            {
+                                                                "label": "",
+                                                                "hidden": true,
+                                                                "tableView": true,
+                                                                "clearOnHide": false,
+                                                                "key": "outstandingWarrants_yn",
+                                                                "type": "textfield",
+                                                                "input": true
+                                                            },
+                                                            {
+                                                                "label": "HTML",
+                                                                "tag": "p",
+                                                                "attrs": [
+                                                                    {
+                                                                        "attr": "style",
+                                                                        "value": "list-style-type:none;"
+                                                                    }
+                                                                ],
+                                                                "content": "The client doesn't have any Outstanding Warrants",
```

Comment:
This is going to be sooo picky and I am sorry that this is how my brain works haha but should Outstanding Warrants be lower case? outstanding warrants? I ask because below you have 'programs' in lower case

Reply:
Nice catch, I made it up, expecting the final sentence from BA team. :) So we can leave it as is, and waiting for the final text. :)

Reply:
okay perfect haha!! Approved